crates/bevy_core_pipeline/Cargo.toml

```diff
@@ -12,6 +12,10 @@ repository = "https://github.com/bevyengine/bevy"
 license = "MIT OR Apache-2.0"
 keywords = ["bevy"]
 
+[features]
+default = []
```

Comment:
Why add an empty default feature?

Reply:
I thought it was necessary. If the default default features is empty then I’ll remove it.

Reply:
Yup by default all features are disabled (aka empty)
https://doc.rust-lang.org/cargo/reference/features.html#the-default-feature

Reply:
I’m not at my computer so if you want to make that change and merge it, please go ahead. :)

Reply:
Done.